package.json

```diff
@@ -38,7 +38,8 @@
     "koa-bodyparser": "^4.1.0",
     "koa-jwt": "^3.0.0",
     "koa-router": "^7.0.0",
-    "mongoose": "^4.8.0"
+    "mongoose": "^4.8.0",
+    "winston": "^2.3.1"
```

Comment:
I would use [bunyan](https://github.com/trentm/node-bunyan) with the [Koa middleware](https://github.com/koajs/bunyan-logger).